grails-app/services/au/org/ala/bie/IndexService.groovy

```diff
@@ -30,18 +29,21 @@ class IndexService implements GrailsConfigurationAware {
     SolrQuery suggestTemplate
     List<String> searchFq
     String searchDefType
+    String searchBoost
     List<String> searchQf
 
     @Override
     void setConfiguration(Config config) {
         def search = config.solr.search
+        searchBoost = search.boost
+        searchDefType = search.defType
         searchTemplate = new SolrQuery()
         search.fq.each { searchTemplate.addFilterQuery(it) }
-        searchTemplate.set('defType', search.defType)
+        searchTemplate.set('defType', searchDefType)
```

Comment:
Do you need to check null or the template works with null?

Reply:
These should never be null, the application.yml provides a default value if nothing is supplied in the configuration.

Reply:
See https://github.com/AtlasOfLivingAustralia/bie-index/blob/master/grails-app/conf/application.yml#L227